aq_cons/www/js/View/Panels/AQCons/AQConsMasterPanelView.js

```diff
@@ -12,19 +13,126 @@ App.View.Panels.AQCons.Master = App.View.Panels.Base.extend({
       id_panel: 'master'
     });
 
-    App.View.Panels.Base.prototype.initialize.call(this,options);
+    let optionsMap = {
+      defaultBasemap: 'positron',
+      center: [-6.0738382, 37.3357641]
+    }
+
+    App.View.Panels.Base.prototype.initialize.call(this, options);
+    this._mapInstance = new App.View.Map.MapboxView(optionsMap);
+    this.listenTo(this._mapInstance.mapChanges,'change',(change)=>{
+      if (change.changed.loaded) {
+        this._onMapLoaded()
+      } else if (change.changed.bbox) {
+        this._onBboxChange(change.changed.bbox);
+      }
+    });
     this.render();
   },
 
-  customRender: function(){
-    //this._widgets = [];
-    //
-    //
-    // //CONTENEDORES EN PELIGRO DE DESBORDAMIENTO
-    // this._widgets.push(new App.View.Widgets.Dumps.ContainerOverflow({
-    //   id_scope: this.scopeModel.get('id'),
-    //   link: '/' + this.scopeModel.get('id') + '/dumps/dashboard/dumps',
-    // }));
-
-  }
+  render: function () {
+    this.subviews.push(this._mapInstance);
+    App.View.Panels.Base.prototype.render.call(this);
+  },
+
+  _onMapLoaded: function() {
+    // Modelos
+    let sensor = new App.Model.Aq_cons.Model({scope: 'aljarafe', entity: 'aq_cata.sensor'});
```

Comment:
Para todos estos, el scope tienes que ser dinámico, normalmente lo tienes disponible en `this.scopeModel.get('id')` cuando heredas un panel.